apps/tup-cms/src/apps/portal_nav/templates/portal_nav/nav_portal.raw.html

```diff
@@ -0,0 +1,54 @@
+{# @var user #}
+<!-- FAQ: This template loads independently at a unique url (see `urls.py`)
+          so CMS and User Guide can render this markup into their markup. -->
+
+{# GH-101: Remove style tag i.e. rely only on `site.header.css` #}
+<style type="text/css">
+  .s-portal-nav .icon {
+    margin-right: 0.25em;
+
+    /* Copied from FontAwesome `.fa-lg` */
+    font-size: 1.3em;
+    vertical-align: text-bottom; /* tweaked to align better */
+  }
+  .s-portal-nav .nav-link .icon {
+    font-size: 1.5em;
+    vertical-align: middle; /* tweaked to align better */
+  }
+</style>
```

Comment:
❗️ Wes... if this was copied from Portal, then would CMS (the host of other containers) even need this?

Reply:
Seems they do... Gross. https://github.com/TACC/Core-CMS/issues/101 would be nice. Or moving this template to core as a single source of truth.